Sources/Swift/Helper/SentryEnabledFeaturesBuilder.swift

```diff
@@ -57,7 +57,18 @@ import Foundation
             features.append("fastViewRendering")
         }
 #endif // #if os(iOS) && !SENTRY_NO_UIKIT
+
+        addFileIOFeatures(options: options, to: &features)
```

Comment:
`l`: IMO, the extra method harms readability a bit. Now I have to jump around to see which features are added to the list. For me, it's OK to have all features in one method. No need to add an extra one.

Reply:
I also don't like this but swiftlint complained about function body length and cyclomatic complexity is already disabled too, so maybe it's worth starting to extract into multiple methods.

Reply:
As we both don't like this I'll change it and force-ignore the linter